### DISCUSSION TITLE
Known Issues

### DISCUSSION THREAD

**Discussion by Akascape** [Akascape/TkNodeSystem#1] [Announcements]
Current Issues
- Lambda functions cannot be saved/loaded properly. [[1]](https://github.com/Akascape/TkNodeSystem/blob/b5908300b66e2d7862f40663a3efe5b8d0ead9c1/tknodesystem/node_types.py#L156)
   If someone knows how to properly save function/command details to a file and reuse it, then let me know.
- ~Recursion problem when one flow output is connected to same flow input.~
- ~Multiple connected operation nodes connected to one output operation node are not updated.~

**Comment by felipelourentino** [comment]
olá, baixei seu projeto, bem legal ele, não rodou... apenas o exemplo simples... penso em colaborar com ele...
